.appveyor.yml

```diff
@@ -0,0 +1,119 @@
+version: "{build}"
+
+stack: go 1.16.7
+
+environment:
+  matrix:
+  # For regular jobs, such as push, pr and etc.
+  - job_name: Linux
+    appveyor_build_worker_image: ubuntu1804
+  # - job_name: MacOS
```

Comment:
This never needs to run on Mac or Windos so we should just remove it imo.

Reply:
Done.